### PR TITLE
LoopTools build_tarballs

### PR DESCRIPTION
**Pull request by fkguo**
this failed..

### CODE REVIEW

File: L/LoopTools/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.feynarts.de/looptools/LoopTools-2.15.tar.gz", "a065ffdc4fe6882aa3bb926134ba8ec875d6c0a633c3d4aa5f70db26542713f2"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/LoopTools-*
+
+./configure --prefix=$prefix --build=${MACHTYPE} --host=${target}
```
Comment (giordano) [inline L/LoopTools/build_tarballs.jl:17]: See https://discourse.julialang.org/t/needs-help-to-build-looptools-using-binarybuilder/44260/6
```suggestion
export AR=ar
export RANLIB=ranlib
./configure --prefix=$prefix --host=${target}
```
and you need to patch `configure` to remove the use of the unsafe flag `-ffast-math`.  With these changes, it should work

**Comment by giordano** [comment]
Now it should work for all platforms

**Comment by fkguo** [comment]
Thank you!

**Comment by fkguo** [comment]
> Now it should work for all platforms

Hello, is it possible to make it also work for Mac m1?

**Pull request merged**